homeassistant/components/onvif/translations/en.json

```diff
@@ -35,7 +35,8 @@
                 "data": {
                     "host": "Host",
                     "name": "Name",
-                    "port": "Port"
+                    "port": "Port",
+                    "ffmpeg_snapshot": "Only use FFMpeg for snapshots"
```

Comment:
Dont forget to add this to strings.json too

Reply:
Good catch! Again, I'm the newby in here!

Reply:
OK, missed something, reopening until fixed.